src/riscv-mode-integration.adoc

```diff
@@ -1,14 +1,7 @@
-[#chapter-Zcheri-mode]
-== "Zcheri_mode" Extension for CHERI Execution Mode
-
-{cheri_mode_ext_name} is an optional extension to {cheri_legacy_ext_name}.
-Implementations that support {cheri_mode_ext_name} allow fine-grained switching
-between Capability and Legacy modes using indirect jump instructions.
-
 [#section-cheri-execution-mode]
-=== CHERI Execution Mode
+=== Capability format
 
-{cheri_mode_ext_name} adds a new CHERI execution mode bit (M) to capabilities.
```

Comment:
We can probably fold this file into riscv-legacy-integration.adoc, but I wanted to keep the changes smaller initially.